ouroboros-consensus/src/Ouroboros/Consensus/MiniProtocol/BlockFetch/Server.hs

```diff
@@ -66,16 +66,17 @@ instance Exception BlockFetchServerException
 -- 'Ouroboros.Network.BlockFetch.Examples.mockBlockFetchServer1', but using
 -- the 'ChainDB'.
 blockFetchServer
-    :: forall m blk.
+    :: forall m blk version.
        ( IOLike m
        , StandardHash blk
        , Typeable     blk
        )
     => Tracer m (TraceBlockFetchServerEvent blk)
     -> ChainDB m blk
+    -> version
```

Comment:
I would just make it the real type (`NodeToNodeVersion blk`) that you're going to be passing to it. I know it now expresses that it doesn't care about the version, but the real type documents better what is to come. Same for the other handlers.

Reply:
I was just worried about testing code, but I think this can be done without any problem.

Reply:
Done